vlib/v/tests/skip_unused/generics_as_receiver.vv

```diff
@@ -0,0 +1,12 @@
+import time
+
+struct Encoder {}
+
+fn (e &Encoder) encode_value_with_level[T](val T) string {
```

Comment:
```suggestion
encode_value_with_level[T](val T) string {
```
will be clearer, since the receiver is misleading here

Reply:
Ok, 👍